lib/mailboxer/mail_dispatcher.rb

```diff
@@ -1,21 +1,18 @@
 module Mailboxer
   class MailDispatcher
+    attr_reader :mailable, :receipts
 
-    attr_reader :mailable, :recipients
-
-    def initialize(mailable, recipients)
-      @mailable, @recipients = mailable, recipients
+    def initialize(mailable, receipts)
+      @mailable, @receipts = mailable, receipts
     end
 
     def call
+      # Why is this here, the dispatch should dispatch email not decide to dispatch
```

Comment:
Please make an issue or a PR for this.  This sort of comment gets confusing in the code.